bitrise.yml

```diff
@@ -9,17 +9,16 @@ pipelines:
 stages:
   stage_1:
     workflows:
-    - configure_build: {}
+    - configure_build_fennec: {}
+    - configure_build_fennec_xcuitest: {}
```

Comment:
I have changed the stages on Bitrise.
Now on the first stage we will build Fennec and Fennec_Enterprise_XCUITests, we need the xctestrun from Fennec for the Unit Tests and the coverage and the xctestrun from Fennenc_Enterprise_XCUITests to run the UI tests.

I tried to use Fennec's xctestrun for all the tests but the UI Tests fail using that configuration. I will create a follow up PR to investigate that since having just one scheme and xctestrun for all the tests and test plans will simplify things in the future.

In the `configure_build_fennec` we build the app, run the unit tests and generate the coverage report. If something goes wrong, the second stage which runs all the UI tests will not run.
The `configure_build_fennec_xcuitest` runs in parallel to have the build ready in case we go to the second stage to execute the UI tests.

Reply:
<img width="397" alt="Screenshot 2023-01-30 at 15 29 24" src="https://user-images.githubusercontent.com/1897507/215505026-8ce0a8d0-03d7-4a61-9417-789bb4759444.png">